Source/Initialization/InjectorMomentum.H

```diff
@@ -225,7 +225,7 @@ struct InjectorMomentumJuttner
         using namespace amrex;
         Real u[3];
         for (int idim = 0; idim < 3; ++idim) u[idim] = 0.0_rt;
-        const Real gamma = 1./sqrt(1+beta*beta);
+        const Real gamma = static_cast<Real>(1./sqrt(1+beta*beta));
```

Comment:
Do we want to perform this calculation always in double and then cast the result? Ok with me, but we can still avoid the implicit `int` cast:
```suggestion
        const Real gamma = static_cast<Real>(1./sqrt(1.+beta*beta));
```

Reply:
I left the calculation in double precision because I thought that this was the intent, but maybe it's accidental. I don't now if performing the calculation in single precision leads to bugs.
Anyway, we should certainly follow your suggestion.